dask/graph_manipulation.py

```diff
@@ -52,11 +52,10 @@ def checkpoint(
     :doc:`delayed` yielding None
     """
     if split_every is None:
-        # FIXME https://github.com/python/typeshed/issues/5074
```

Comment:
Was this special-cased in mypy? I don't see any linked issues in the above. In any event, nice to remove `type: ignore`s